services/dns_ssl_service/app.py

```
@@ -0,0 +1,68 @@
+@LAT.labels('ssl').time()
+def ssl_info():
+	REQS.labels('ssl').inc()
+	host = request.args.get('host', '')
+	port = int(request.args.get('port', '443'))
+	if not host:
+		ERRS.labels('ssl').inc()
+		return jsonify({'error': 'host required'}), 400
+	try:
+		context = ssl.create_default_context()
+		with socket.create_connection((host, port), timeout=5) as sock:
+			with context.wrap_socket(sock, server_hostname=host) as ssock:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)